user/gnome-podcasts/template.py

```diff
@@ -0,0 +1,55 @@
+pkgname = "gnome-podcasts"
+pkgver = "0.7.1"
+pkgrel = 0
+build_style = "meson"
+hostmakedepends = [
+    "cargo-auditable",
+    "desktop-file-utils",
+    "gettext",
+    "meson",
+    "ninja",
+    "pkgconf",
+]
+makedepends = [
+    "dbus-devel",
+    "gst-plugins-bad-devel",
+    "gstreamer-devel",
+    "libadwaita-devel",
+    "openssl-devel",
+    "rust-std",
+    "sqlite-devel",
+]
+pkgdesc = "GTK-based podcast aggregator"
+maintainer = "tulilirockz <tulilirockz@outlook.com>"
+license = "GPL-3.0-or-later"
+url = "https://gitlab.gnome.org/World/podcasts"
+source = f"{url}/-/archive/{pkgver}/podcasts-{pkgver}.tar.gz"
+sha256 = "933d33dfd0f36343f9c80f055a48e14307a0665b35097da176767ddbfe583399"
+# check: broken tests due to sandboxing
+options = ["!check"]
+
+
+def post_patch(self):
+    from cbuild.util import cargo
+
+    cargo.Cargo(self, wrksrc=".").vendor()
+
+
+def init_build(self):
+    from cbuild.util import cargo
+
+    renv = cargo.get_environment(self)
+    self.make_env.update(renv)
+
+
+def pre_build(self):
+    self.mkdir(self.cwd / "build/podcasts-gtk/src/release", parents=True)
+    (self.cwd / "build/podcasts-gtk/src/release/podcasts-gtk").touch(0o755)
+
+
+def post_install(self):
+    self.uninstall("usr/bin/gnome-podcasts")
+    self.install_bin(
+        f"build/podcasts-gtk/src/{self.profile().triplet}/release/podcasts-gtk",
+        name=pkgname,
+    )
```

Comment:
look at the other cargo meson things and delete the meson that installs this and manually do it instead of these hacks

Reply:
sorry, i literally did not know this was possible LOL, after seeing the loupe approach it makes so much sense